static/src/javascripts-legacy/projects/common/modules/experiments/tests/paid-content-vs-outbrain.js

```diff
@@ -24,12 +24,20 @@ define([
         variants: [
             {
                 id: 'paid-content',
-                test: function () {
+                // TODO: ask someone about this
```

Comment:
I'll try this in a separate PR and you can see what you think - there are some other bits that would need to come out as well if we remove this file (e.g. https://github.com/guardian/frontend/blob/044631fb74fbadf083fc512cd197090fd675e9af/static/src/javascripts/projects/commercial/modules/high-merch.js#L12-L14) so it might not be worth the hassle of re-instating them all later